services/addons/tests/firmware_manager/test_download_blob.py

```diff
@@ -6,51 +6,6 @@
 from addons.images.firmware_manager.download_blob import UnsupportedFileTypeException

```

Comment:
This class was removed but the import statement remains; running into an error when running unit tests due to this

Reply:
My mistake, for some reason my vscode wasn't picking up that failing unit test.